tox.ini

```
@@ -41,7 +41,7 @@ commands =
     # Clean up from previous tests
     python util.py clean-tests
     # Run tests
-    pytest -vv -rs --cov=sqlfluff --cov-report=xml {posargs: test}
+    pytest -n 2 -vv -rs --cov=sqlfluff --cov-report=xml {posargs: test}
```

@pwildenhain we have posargs on the tox tests so lets set this line back to no multitasking by default
```suggestion
    pytest -vv -rs --cov=sqlfluff --cov-report=xml {posargs: test}
```

and then in `.github/workflows/ci-tests.yml` line 64 (the tox tests for the standard python unit tests) pass `-n 2` via posargs:
i.e.
```
tox -e ${{ env.TOXENV }} -- -n 2
```

This will use the multitasking on the python tests only and leave the dbt ones as they were before, which should hopefully let us get the speed boost on the main tests without impacting the dbt ones 😄

---

FYI posargs (the `--`) works by replacing the contents of `{posargs: test}` (where `test` is the default value for no posargs).
So in this case its like running `pytest -vv -rs --cov=sqlfluff --cov-report=xml -n 2 test` for the standard tests
and `pytest -vv -rs --cov=sqlfluff --cov-report=xml plugins/sqlfluff-templater-dbt` for the dbt tests 👍